pKExample/build.gradle

```diff
@@ -1,13 +1,13 @@
-apply plugin: 'android'
+apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion "20.0.0"
+    compileSdkVersion 26
+    buildToolsVersion "27.0.3"
 
     defaultConfig {
         applicationId "com.brendan.pkexample"
-        minSdkVersion 8
-        targetSdkVersion 19
+        minSdkVersion 19
+        targetSdkVersion 26
```

Comment:
same

Reply:
because I updated support lib versions there. I will make it here as well for consistency